src/include/enclave/lthread_int.h

```diff
@@ -71,4 +71,20 @@ static inline uint64_t _lthread_timespec_to_usec(const struct timespec* ts)
     return (ts->tv_sec * 1000000) + ts->tv_nsec / 1000;
 }
 
+// TODO: should this be static?
+static inline struct schedctx *__scheduler_self()
```

Comment:
Please add a doc comment.  It should be `static inline` so that it can be inlined everywhere (it's a single instruction after inlining) and the compiler won't complain about it being in a header.

Reply:
Added docs.